week03/inho/java-christmas-6/src/main/java/christmas/model/Event.java

```diff
@@ -0,0 +1,24 @@
+package christmas.model;
+
+import christmas.constant.event.EventName;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import static christmas.view.OutputView.formatPrice;
+
+public class Event {
+
+    protected int discountPrice;
+    protected EventName eventName;
+    private static final int DISCOUNT_MULTIPLIER = -1;
+
+    public int getDiscountPrice() {
+        return discountPrice * DISCOUNT_MULTIPLIER;
+    }
+
+    @Override
+    public String toString() {
+        return eventName.getName() + ": " + formatPrice(discountPrice * DISCOUNT_MULTIPLIER) + "원";
+    }
+}
```

Comment:
```suggestion
package christmas.model;

import christmas.constant.event.EventName;

import static christmas.view.OutputView.formatPrice;

public class Event {

    protected int discountPrice;
    protected EventName eventName;
    private static final int DISCOUNT_MULTIPLIER = -1;

    public int getDiscountPrice() {
        return discountPrice * DISCOUNT_MULTIPLIER;
    }

    @Override
    public String toString() {
        return eventName.getName() + ": " + formatPrice(discountPrice * DISCOUNT_MULTIPLIER) + "원";
    }
}
```

사용하지 않는 import문은 제거해주시면 좋을 것 같아요!

Reply:
추가로 OutputView에 의존성이 생기는 것도 다시 한번 고민해볼 필요가 있을 것 같습니다!

Reply:
view랑 model 의존성 분리시키겠습니다!